admin_manual/installation/source_installation.rst

```diff
@@ -2,47 +2,177 @@
+   - Includes Office, a turnkey Backup solution, Imaginary (for previews of heic, heif, illustrator, 
+     pdf, svg, tiff and webp) and much more (many components are optional). Full feature list available
+     `here <https://github.com/nextcloud/all-in-one?tab=readme-ov-file#nextcloud-all-in-one>`_.
+   - Up to 100 users are free (though without any direct service or support or guarantees). Free community 
+     support is provided via the `Nextcloud Community Help Forum <https://help.nextcloud.com/>`_ and a 
+     `dedicated GitHub Discussions area <https://github.com/nextcloud/all-in-one/discussions>`_ (on a best 
+     efforts basis).
+   - For slightly larger installations (>100 users) an AIO specific `Nextcloud Enterprise arrangement 
+     <https://nextcloud.com/all-in-one/>`_ is required (it includes a support SLA, security information and 
+     all the other benefits that come with `Nextcloud Enterprise <https://nextcloud.com/all-in-one/>`_).
+   - `Official Documentation <https://github.com/nextcloud/all-in-one>`_ (**Note**: see Official Documentation 
+     for AIO specific installation and usage details).
+* Nextcloud Community Snap
+   - A community-driven installation method that is actively maintained by amazing Nextcloud community members 
+     (make sure to thank them - or find a way help them out - if you get the opportunity!).
+   - A full production-ready stack that will maintain your HTTPS certificates for you and will automatically 
+     update as needed to stay secure. Full feature list available 
+     `here <https://github.com/nextcloud-snap/nextcloud-snap?tab=readme-ov-file#snappy-nextcloud>`_.
+   - Designed specifically for Ubuntu (`non-Ubuntu distributions are not officially supported 
+     <https://github.com/nextcloud-snap/nextcloud-snap/wiki/Why-Ubuntu-is-the-only-supported-distro>`_).
+   - `Official Wiki <https://github.com/nextcloud-snap/nextcloud-snap/wiki>`_
+   - `Official Documentation <https://github.com/nextcloud-snap/nextcloud-snap>`_ (**Note**: see Official 
+     Documentation - and Wiki - for Snap specific installation and usage details).
+* Nextcloud Community VM Appliance (aka Nextcloud Virtual Machine or NcVM)
+   - A community-driven installation method that is actively maintained by amazing Nextcloud community members 
+     (make sure to thank them - or find a way to help them out - if you get the opportunity!) and partner Hansson
+     IT.
+   - Helps you create a personal or corporate Nextcloud Server faster and easier. 
+   - Can be used install directly on a clean Ubuntu Server (as an install script) or downloaded as a fully 
+     functioning VM. Full feature list available `here 
+     <https://github.com/nextcloud/vm/?tab=readme-ov-file#server-installation-simplified-cloud>`_.
+   - `Official Documentation <https://github.com/nextcloud/vm/>`_ (**Note**: see Official Documentation 
+     for VM specific installation and usage details).
+* NextcloudPi (aka: NCP)
```

Comment:
Perhaps we should mention the extras NCP offers to manage the Linux system to be run in "home networks" (dynDNS, letsencrypt, backup etc.).

I see this as a real advantage for home-users to put a device in their local network. But on a public server, I wouldn't use it.